### PR TITLE
README syntax highlighting

### PR DESCRIPTION
**Pull request by hkareem2222**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,9 @@ The package manager is bundled with the [downloads available at swift.org](https
 
 If you want to verify you already have the package manager installed, enter the following in a terminal:
 
-    swift build --help
-
```
Comment (LinusU) [inline README.md]: Why remove this blank line?

File: README.md [hunk 2]
```diff
@@ -27,8 +27,9 @@ The package manager is bundled with the [downloads available at swift.org](https
...
+```bash
```
Comment (LinusU) [inline README.md]: How about using `sh` instead of `bash` since there is nothing bash-specific here

**Comment by hkareem2222** [comment]
@LinusU updated.

File: README.md [hunk 3]
```diff
@@ -44,9 +46,11 @@ For development purposes, the package manager can be built in two ways:
 
 2. To build against a [downloaded](https://swift.org/download/) Swift package, enter the following in a terminal:
 
-        git clone git@github.com:apple/swift-package-manager.git swiftpm
-        cd swiftpm
-        ./Utilities/bootstrap --swiftc path/to/swiftc --sbt path/to/swift-build-tool
+```bash
```
Comment (LinusU) [inline README.md]: `sh` here as well

File: README.md [hunk 4]
```diff
@@ -136,8 +140,9 @@ simple example could be:
 
 If you then run the following command in the directory `foo`:
 
-    $ swift build
-
+```bash
```
Comment (LinusU) [inline README.md]: `sh` here as well

File: README.md [hunk 5]
```diff
@@ -136,8 +140,9 @@ simple example could be:
...
+$ swift build
+```
```
Comment (LinusU) [inline README.md]: Empty line below this line

File: README.md [hunk 6]
```diff
@@ -27,8 +27,10 @@ The package manager is bundled with the [downloads available at swift.org](https
 
 If you want to verify you already have the package manager installed, enter the following in a terminal:
 
-    swift build --help
+```sh
+swift build --help
 
+```
```
Comment (LinusU) [inline README.md]: Please move this line one line up

**Comment by LinusU** [comment]
@hkareem2222 more comments

**Comment by hkareem2222** [comment]
@LinusU how's that?

**Comment by LinusU** [comment]
Perfect :ok_hand:

File: README.md [hunk 7]
```diff
@@ -136,7 +140,9 @@ simple example could be:
 
 If you then run the following command in the directory `foo`:
 
-    $ swift build
+```sh
+$ swift build
+```
```
Comment (davidcelis) [inline README.md]: Weird that none of the other shell commands include a `$` prompt. I'd get rid of it here to be consistent

Comment (hkareem2222) [inline README.md]: makes sense, i'll update it.

**Comment by hkareem2222** [comment]
thanks @davidcelis updated.

File: README.md (hunk 7 above)
Comment (LinusU) [inline README.md]: yes, nice :ok_hand:

**Comment by mxcl** [comment]
Good changes. Appreciated 👍🏻

**Pull request merged**